waspc/src/Wasp/Util.hs

```diff
@@ -270,3 +271,9 @@ secondsToMicroSeconds = (* 1000000)
 
 findDuplicateElems :: Ord a => [a] -> [a]
 findDuplicateElems = map head . filter ((> 1) . length) . group . sort
+
+validateToEither :: (Monad m) => (a -> m [String]) -> a -> m (Either [String] a)
```

Comment:
I'm not sure what to name this ... it takes a validation function that returns a list of errors and based on that either returns the input or the errors. 

`listValidationToEither`, `validateListToEither`, `runValidation`, `listValidationToEither`?

Reply:
Inlined the function 👍